api-gateway/src/main/resources/static/scripts/bill-details/bill-details.template.html

```diff
@@ -18,4 +20,8 @@ <h3>Date generated: {{$ctrl.bills.date}}</h3>
     </div>
     <br>
     <a class="newBtn btn btn-primary" href="http://localhost:8080/#!/bills">Back to History</a>
+    <br><br>
+
+    <a class="newBtn btn btn-primary" href="http://localhost:8080/#!/bills/customerId/{{$ctrl.bills.customerId}}">
```

Comment:
[nitpick] the way to redirect in angular js is to use uisref such as
```
<a ui-sref="btnNew">

   <button class="...">New Button</button>

</a>
```
and add the uisref link to the bill-details.js file as a .state configuration like this
```
      .state('newBtn', {         
    
                parent: 'app',

                url: '/owners/details/{{bill.customerId}}',

                template: '<bills-by-owner-id></bills-by-owner-id>'

         })
```